### DISCUSSION TITLE
機器學習技術製作合成酶，24 小時分解塑膠垃圾

### DISCUSSION THREAD

**Discussion by Yao900108** [1101-GS4719-C/Python-Community-GS4719-C#590] [科技新知]
科學界近年應用人工智慧技術的場合越來越多，最近一個新研究，就利用機器學習技術找出最有效的合成酶成分，再用於分解塑膠垃圾。
美國德州大學奧斯汀分校的科學家早前對 PET 酶的天然結構進行研究，用於可降解聚乙烯塑膠廢料。為了找出最高效率的合成酶，他們訓練了一個機器學習模型，結果從中挑選出一個名為 FAST-PET 的酶進行合成。據稱 FAST-PET 酶可以在攝氏 30 至 50 度的溫度下、24 小時內分解塑膠。團隊希望人工智慧設計的酶生產和使用可以擴大到工業標準，進一步處理全球的塑膠垃圾問題。

https://technews.tw/2022/05/10/enzymes-that-break-down-plastic/

**Comment by JSHT** [comment]
現在有越來越多種微生物能夠降解各類塑膠製品，期望能夠對這些生物的酶進行更深度的研究，再透過科學方法找出跟 FAST-PET 酶一樣有效率的合成酶(對其餘塑料分子)

**Comment by sys511613** [comment]
這個實驗透過機器學習幫助人類解決塑膠垃圾的問題。希望未來能看到更多機器學習結合不同領域的應用~

**Comment by Jay-722** [comment]
現在塑膠垃圾到處都是，的確是一個需要關注的議題

**Comment by handsomeniko** [comment]
太厲害了，謝謝分享

**Comment by Sazmafuji** [comment]
每當看到有塑膠袋飄在河邊，會有點心虛，希望可以解決

**Comment by 110409538** [comment]
每年製造非常多塑料垃圾，這項技術可以幫助更快分解塑料垃圾。

**Comment by lx3211** [comment]
恭喜塑膠不再是燒了有毒氣、埋了是萬年垃圾的廢物了

**Comment by cordydeng** [comment]
這個技術也太厲害了吧，希望可以有效解決垃圾問題

**Comment by 11timyyy** [comment]
雖說科技進步，但人類還是需要減少使用塑膠製品!

**Comment by d26488399** [comment]
太棒了，能更好得處理塑膠垃圾

**Comment by leon10456** [comment]
太厲害了，謝謝分享

**Comment by Rosancl** [comment]
希望能有效解決垃圾問題~~

**Comment by henrywen547** [comment]
人類還是應減少對一次性塑膠的依賴，製造速度比分解速度還快也不會有所效果。

**Comment by stan890425** [comment]
這樣就不用人工判定+合成了

**Comment by liangyinglly** [comment]
他們一定能獲得環保人士們的愛戴

**Comment by HaoFish123** [comment]
感謝分享，但感覺垃圾問題如果因此解決，只是治標不治本。

**Comment by treeann** [comment]
沒想到機器學習可以運用在這方面!

**Comment by alexthebigpp** [comment]
當今人類真的急需這種科技，希望可以進一步發展。

**Comment by bloodtrail0511** [comment]
感謝分享，是個能影響世界的技術呢

**Comment by etin2468** [comment]
能夠高效率解決環境問題，比人工開發感覺快很多

**Comment by Lynn12378** [comment]
可以改善一直以來環保遇到的大難題，塑膠在海洋無法分解，環境保護有了很大的進步。

**Comment by Jimmy911218** [comment]
如果用ai感覺會提高製造的效率
